volto/src/addons/volto-chart-builder/src/reducers/index.js

```diff
@@ -65,3 +70,55 @@ export function chartBuilderRawData(state = initialState, action = {}) {
       return state;
   }
 }
+
+const initialAddItemBlockDataState = {
```

Comment:
If we have multiple add item blocks in a page, it won't be clear which block these flags refer to.

Amend the state so that it's a map of content ID to all these flags, and then each AddItem block can refer to a unique set of loading / error flags

Reply:
Done